pkg/gui/presentation/commits.go

```diff
@@ -130,6 +130,9 @@ func GetCommitListDisplayStrings(
 	lines := make([][]string, 0, len(filteredCommits))
 	var bisectStatus BisectStatus
 	willBeRebased := markedBaseCommit == ""
+	alwaysShowCommitIcon := allOf(filteredCommits, func(c *models.Commit) bool {
```

Comment:
`lo.EveryBy` Does the trick. But do we need to infer this? I'd pass an arg saying whether to always show the icon, and only pass it as true from the sub-commits view when it showing left/right commits

Reply:
> `lo.EveryBy` Does the trick.

Nice; I did search for something in `lo`, but couldn't find it.

> But do we need to infer this? I'd pass an arg saying whether to always show the icon, and only pass it as true from the sub-commits view when it showing left/right commits

Yes, I considered that, but was too lazy to pass it all the way through...